dev/packages/alpha/nginx/0.0.3/dataset/access/agent/stream/stream.yml.hbs

```diff
@@ -0,0 +1,8 @@
+input: log
```

Comment:
I think this should be logs to match the inputs definition.

Reply:
Hmm, I'm not sure about this. Can you take a look at this? https://github.com/elastic/package-registry/blob/master/dev/packages/example/nginx/0.0.2/dataset/access/agent/stream/stream.yml.hbs#L2